### ISSUE TITLE
Where is the source of this binary img?

### ISSUE THREAD

**Issue by w-A-L-L-e** [Amibian/amibian#3]
This github repo only contains a packaged .img file of the complete amibian distro.
However what if I want to make changes and fork or improve? Where can one obtain the build scripts, kernel and package sources used, init scripts etc? Basically one of the great features I notice is the ultra fast boot times (faster than other bare bones raspbian images around). How was the sub 3 second boot achieved and how can it be replicated for other/future projects? Right now this seems hard to do other than just using a big binary blob .img and making changes. Did you use yocto or buildroot ?

Isn't it obligated to have the sources available somewhere because this is a linux + other opens source projects combined isn't it?

Apart from that, keep up the great work! This is a super nice project and keeping the Amiga spirit alive.

**Comment by rzr** [comment]
are there any other amiga linux distro to look at ?

**Comment by gunkrist** [comment]
Hey, didn't see your original email.  Img is a copy of the sd card, similar
to a zipfile if you will... Amibian 1.4 which you are referring to is based
on debian jessie, all sources are included in the img file.
Achieving the ultra fast boot is made by tweeking the linux boot process,
basically rearranging the order of the startup process. Doing that has its
downsides, like if you start a process that requires another process that
has not started yet, therefore it may not be possible for other projects.
There are probably other amiga distros for the Raspberry pi out there, i
guess google knows more about them than i do.
Best regards. Gunnar

**Comment by w-A-L-L-e** [comment]
Thanks for the reply. So in essence starting with a debian jessie and tweaking it would make it possible to boot equally fast. Meanwhile also bought a second hand A500 to refurbish as a hobby project ;).

**Comment by gunkrist** [comment]
Yeah, jessie can start very fast, i havnt gotten as fast boots in debian
buster without some problems.
Best regards.Gunnar
